xapian-applications/omega/omega.cc

```diff
@@ -364,6 +364,45 @@ try {
 	}
     }
 
+    // Set auto boost terms
+    g = cgi_params.equal_range("AUTOBOOST");
+    if (g.first != g.second) {
+	vector<string> auto_boost;
+	for (auto i = g.first; i != g.second; ++i) {
+	    const string& v = i->second;
+	    if (!v.empty() && C_isalnum(v[0])) {
+		add_boost_term(v);
+		auto_boost.push_back(v);
+	    }
+	}
+	sort(auto_boost.begin(), auto_boost.end());
+	vector<string>::const_iterator j;
+	for (j = auto_boost.begin(); j != auto_boost.end(); ++j) {
+	    const string& boost_term = *j;
```

Comment:
Moreover, since filters are mostly used for reloading if the query has changed. Is it necessary to add a symbol to distinguish terms added with `AUTOBOOST` and with `B`? I don't think it's required but would be great to know your views.